lib/auth/init.go

```diff
@@ -264,6 +272,10 @@ func Init(cfg InitConfig, dynamicConfig bool) (*AuthServer, *Identity, error) {
 		}
 	}
 
+	if cfg.DebugMode {
+		log.Warn("[INIT] Starting Teleport in Debug mode.")
```

Comment:
should also communicate implications of this - (origin check off/insecure TLS, etc) so folks know what are they getting